FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -131,20 +141,17 @@ lemma intValuation_comap (hAB : Function.Injective (algebraMap A B))
   simp
 
 -- Need to know how the valuation `w` and its pullback are related on elements of `K`.
-omit [IsIntegralClosure B A L] [FiniteDimensional K L] [Algebra.IsSeparable K L] in
 lemma valuation_comap (w : HeightOneSpectrum B) (x : K) :
-    (comap A w).valuation x ^
-    (Ideal.ramificationIdx (algebraMap A B) (comap A w).asIdeal w.asIdeal) =
-    w.valuation (algebraMap K L x) := by
-  obtain ⟨x, y, hy, rfl⟩ := IsFractionRing.div_surjective (A := A) x
-  simp [valuation, ← IsScalarTower.algebraMap_apply A K L, IsScalarTower.algebraMap_apply A B L,
-    ← intValuation_comap A B (algebraMap_injective_of_field_isFractionRing A B K L), div_pow]
+    w.valuation (algebraMap K L x) = (comap A w).valuation x ^
+    (comap A w).asIdeal.ramificationIdx (algebraMap A B) w.asIdeal := by
+  -- This should follow from map_comap?
+  sorry
```

Comment:
Why are you deleting a completed proof and replacing it with a `sorry`?

Reply:
Whoops, looks like a bad merge